app/jobs/quarterly_notifications_job.rb

```diff
@@ -12,130 +12,38 @@ class QuarterlyNotificationsJob < CaseflowJob
   #
   # Params: none
   #
-  # Response: None
-  def perform # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/PerceivedComplexity
+  # Response: SendNotificationJob queued to send_notification SQS queue
+  def perform
     ensure_current_user_is_set
 
     AppealState.where.not(
       decision_mailed: true, appeal_cancelled: true
     ).find_in_batches(batch_size: QUERY_LIMIT.to_i) do |batched_appeal_states|
```

Comment:
```suggestion
    AppealState.where(
      decision_mailed: false, appeal_cancelled: false
    ).find_in_batches(batch_size: QUERY_LIMIT.to_i) do |batched_appeal_states|
```
What do you think about removing the double negative here? I think `.where(...).not` is being deprecated in Rails 6.1, also.